contributing_to_docs/contributing.adoc

```diff
@@ -8,10 +8,14 @@
 
 toc::[]
 
+[NOTE]
+====
+Are you contributing content for 4.0? If yes, please read how to xref:contributing_user_stories.adoc[contribute user stories].
+====
+
 == Different ways to contribute
 There are a few different ways you can contribute to OpenShift documentation:
 
-* Submit comments at the bottom of each topic (still awaiting implementation)
 * Email the OpenShift documentation team openshift-docs@redhat.com
```

Comment:
Is it important to keep email as an option? I think it would be helpful to remove this extra channel to monitor, if possible.

Reply:
:) I think so at this stage till DDF becomes a reality.